src/components/dashboard-shortcut/dashboard-shortcut.tsx

```diff
@@ -0,0 +1,55 @@
+import { Component, Prop, h } from '@stencil/core';
+
+/**
+ * @exampleComponent limel-example-dashboard-shortcut
+ */
+
+@Component({
+    tag: 'limel-dashboard-shortcut',
+    shadow: true,
+    styleUrl: 'dashboard-shortcut.scss',
+})
+export class DashboardShortcut {
+    /**
+     * Name of icon for the shortcut.
+     */
+    @Prop()
+    public icon: string;
+
+    /**
+     * The text to show below the shortcut. Long label will be truncated.
+     */
+    @Prop({ reflect: true })
+    public label: string;
+
+    /**
+     * Set to `true` if shortcut is disabled.
+     */
+    @Prop({ reflect: true })
+    public disabled: boolean;
+
+    /**
+     * The url that the shortcut leads to.
+     */
+    @Prop({ reflect: true })
+    public href?: string;
+
+    public render() {
+        return [
+            <a
+                aria-disabled={this.disabled}
+                onClick={this.handleClick}
+                href={this.href}
+                tabindex="0"
+            >
+                <limel-icon name={this.icon} />
+            </a>,
+            <span class="label">{this.label}</span>,
+        ];
+    }
+
+    private handleClick = (event: MouseEvent) => {
+        event.stopPropagation();
```

Comment:
We should not stop this click.

Reply:
The easiest way to handle it is probably just to let the native click through. If there's an `href`, the browser will automatically activate that, unless the consumer handles the click event differently, and if there isn't an `href`, the consumer has to handle the click event somehow, or nothing at all will happen 🙂

Reply:
We should document the event though, so the consumer knows to expect it.

Reply:
Thanks Adrian. How do we make sure that the onClick triggers when the `a` is clicked, not when the whole component is clicked?

Reply:
In this specific case, since we're talking about "dashboard shortcut", I'll start by answering your question with another question: shouldn't it trigger regardless of where on the component you click? 🤔

But assuming you have a good answer for that, or just in the more general case: that's a good point! 👍

An alternative would be to have an `interact` event on this component, catch the click event from the link, and emit our custom interact event instead. But we'd have to make sure we included the original click event in the details for the interact event, because the consumer might provide a href for the link, and still wish to preventDefault on the click and handle the navigation themselves. For example, we would like to always provide a url to all our links, so the user can right-click or cmd-click to open the link in a new window, but if we detect a straight left-click with no modifier keys active, we might want to preventDefault on it, and trigger the navigation through the router, because if we just let the link trigger the navigation, it makes the app load again before you get to the link's destination 🙁

Reply:
I'm happy to "consult" on this component, as always 😊 But now I'll try to log off to have me a spot of weekend 😄

# 👋 ❤️

Reply:
Thanks a lot! I think I'll need your help next week. 
But quick answer: The interactive part of this component is the link part (the `<a>`) so I definitely think we should make keyboard or pointer actions only work on that, not the entire component's surface.

Reply:
But it's basically meant to be an app icon on the start page, like on the home screen of Android or iOS, right?

![image](https://user-images.githubusercontent.com/154725/197210122-496bcd83-3e38-4bd6-943a-783447b715cb.png)

You wouldn't want to have to click or tap on the text to trigger the link, right?

Although, my own example shows that there could be situations where a click should not trigger the link, like if you've long-pressed an icon on iOS and entered the edit mode where you can tap the x-button to remove a shortcut… So yes, I agree that we need to be able to not trigger the link from just _any_ click on the component. But still, in the _normal_ case, surely the whole thing should be a click target and trigger the corresponding "action", whatever it may be?

Reply:
@adrianschmidt I think we're ready for review, except the part for the `onClick`. What do you think is best to do here?

Reply:
Nice! I'll have a look at it when I'm done with some stuff I'm working on at the moment 😊

Reply:
Cool. Meanwhile, I'll use this component in crm-components instead of what we have there in `action-button` and connect it the other PR to this issue.

Reply:
ping me before you start, because until then I'll be forcepushing my fixups

Reply:
I'm starting on it now 👍